components/MembershipDriveRoadmap/MembershipDriveRoadmap.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import style from "../../styles/MembershipDrivepage/MembershipDriveRoadmap.module.css";
+
+export default function MembershipDriveRoadmap(props) {
```

Comment:
destructure the props instead of using props.x everywhere

Reply:
Sure